R/EMODnetWFS-package.R

```diff
@@ -6,6 +6,20 @@
 ## usethis namespace: end
 NULL
 
+emodnetwfs_collaborators <- function() {
+  readLines(system.file("collaborators.txt", package = "EMODnetWFS"))
+}
+
 emodnetwfs_user_agent <- function() {
-  "EMODnetWFS R package https://github.com/EMODnet/EMODnetWFS"
+  version <- as.character(packageVersion("EMODnetWFS"))
+
+  if (nzchar(Sys.getenv("EMODNETWFS_CI"))) {
+  	return(sprintf("EMODnetWFS R package %s CI https://github.com/EMODnet/EMODnetWFS", version))
+  }
+
+  if (whoami::gh_username() %in% emodnetwfs_collaborators()) {
```

Comment:
I see an error when using this when I have not set one of GITHUB_USERNAME or GITHUB_PAT. I have, however, the email available. It stops at `lookup_gh_username(email, token)`

> Error in fallback_or_stop(fallback, "Cannot find GitHub username for email") : 
>   object 'fallback' not found

Fun: `whoami::gh_username()`
```r
function (token = NULL, fallback = NULL) 
{
  env_gh_username <- Sys.getenv("GITHUB_USERNAME")
  if (nzchar(env_gh_username)) 
    return(env_gh_username)
  email <- try(email_address(), silent = TRUE)
  if (ok(email)) {
    if (!grepl("@", email)) {
      return(fallback_or_stop(fallback, "This does not seem to be an email address"))
    }
    lookup_gh_username(email, token)
  }
  else {
    fallback_or_stop(fallback, "Cannot get GitHub username")
  }
}
```

@maelle is this something to be reported or am I using it wrong?

Reply:
what does `whoami::email_address()` return?

Reply:
in any case it was bad that I hadn't added a fallback, now done.

Reply:
returns my VLIZ email